server/routing/metadataedit.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express");
+const router = express.Router();
+const path = require("path");
+const exiftool = require("node-exiftool");
+const exiftoolBin = require("dist-exiftool");
+const ep = new exiftool.ExiftoolProcess(exiftoolBin);
+
+const getFileList = require("../helper");
+const filePattern = ".(jpg|jpeg)";
+
+/* stolen from here: https://coderwall.com/p/urxpsa/remove-falsy-values-or-empty-strings-from-javascript-objects */
```

Comment:
Habe für sowas auch einen "schönen" einzeiler gefunden. Auch einfach mal reingeworfen :) Finde ich ganz nett weil man "ordentlich" die array.filter()-Funktion nutzen kann. So muss man kein forEach starten. Aber - nur ein FYI 👍🏻 

```js
const VALIDFIELDS = ['Some Key', 'Some Other Key', 'usw'];
const fullObject = { /* some huge object */};
const filteredObject = Object.fromEntries(Object.entries(fullObject).filter((entry) => VALIDFIELDS.indexOf(entry[0]) > -1));
```

Kurz gesagt:
1. Objekt in seine Einträge zerlegen
2. Die Einträge filtern
3. Objekt wieder zusammenbauen

Reply:
Schau ich mir später genauer ein. Danke!